clang/include/clang/CConv/ConstraintBuilder.h

```diff
@@ -14,6 +14,9 @@
 
 #include "ProgramInfo.h"
 
+typedef std::map<CallExpr *, std::map<unsigned int, const clang::Type *>>
```

Comment:
Add comment to explain this map. Your PR describes why it has this form and what it's used for, so borrow from that?

Reply:
Here you have the type end with capital `T`; further down you have it end with `Type`. I prefer the former. Is there a clang style guide standard for this?

Reply:
Nothing in the style guide as far as I can tell. We're also inconsistent within our own code. I'll make it `T`.